Makefile

```
@@ -60,6 +82,73 @@ zephyr-setup: fprime-venv ## Set up Zephyr environment
+		elif git apply --reverse --check "$$p" 2>/dev/null; then \
+			echo "⚠ Already applied: $$name"; \
+		else \
+			echo "❌ Cannot apply $$name — check usp_zephyr revision"; exit 1; \
+		fi; \
+	done
+
+# USP_DIR: the Semtech smtc_rac_lib west module (radio planner lives here).
+USP_DIR ?= $(shell pwd)/lib/zephyr-workspace/modules/lib/usp
+
+.PHONY: usp-core-patches
+usp-core-patches: ## Apply usp (smtc_rac_lib) patches (radio-planner failsafe unlock exemption)
```

checkmake (0.3.2)

[warning] Target body for "usp-core-patches" exceeds allowed length of 5 lines (11).

(maxbodylength)